learned.lic

```diff
@@ -17,9 +17,16 @@ DRSkill.list.select { |item| item.current - item.baseline > 0 }.sort_by { |item|
   respond(output)
 end
 
+
+
 total = DRSkill.list.select { |item| item.current - item.baseline > 0 }.map { |skill| DRSkill.gained_exp(skill.name) }.inject(&:+).round(2)
+tdps = DRSkill.list.select { |item| item.current - item.baseline > 0 }.map { |skill| (DRSkill.gained_exp(skill.name) * DRSkill.getrank(skill.name))/200 }.inject(&:+).round(2)
```

Comment:
Why does it `select` on `item.current - item.baseline > 0`? It is possible to lose TDPs if you die.